src/shared/nt-results/ArrowHeadMarker.react.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import styles from './ArrowHeadMarker.styl'
+
+export const WIDTH = 2
+export const HEIGHT = 3
+
+class ArrowHeadMarker extends React.Component {
+  render () {
+    return (
+      <marker
+        id='arrowHead'
+        viewBox={`0 ${-(HEIGHT / 2)} ${WIDTH} ${HEIGHT}`}
+        refX={WIDTH / 2}
+        orient='auto'
+        markerWidth={WIDTH}
+        markerHeight={HEIGHT}
+      >
+        <path
+          className={styles.nt}
+          d={`M 0,${-(HEIGHT / 2)} L ${WIDTH},0 L 0,${HEIGHT / 2}`}
+        />
+      </marker>
+    )
+  }
+}
+
+export default ArrowHeadMarker
```

Comment:
Anyways, I decided to use class because something might be changed soon. It is easier to add more code in class.